docs/resources/Stage_Instance.md

```diff
@@ -6,12 +6,21 @@ A _Stage Instance_ holds information about a live stage.
 
 ###### Stage Instance Structure
 
-| Field      | Type      | Description                                        |
-| ---------- | --------- | -------------------------------------------------- |
-| id         | snowflake | The id of this Stage instance                      |
-| guild_id   | snowflake | The guild id of the associated Stage channel       |
-| channel_id | snowflake | The id of the associated Stage channel             |
-| topic      | string    | The topic of the Stage instance (1-120 characters) |
+| Field                 | Type      | Description                                        |
+| --------------------- | --------- | -------------------------------------------------- |
+| id                    | snowflake | The id of this Stage instance                      |
+| guild_id              | snowflake | The guild id of the associated Stage channel       |
+| channel_id            | snowflake | The id of the associated Stage channel             |
+| topic                 | string    | The topic of the Stage instance (1-120 characters) |
+| privacy_level         | integer   | The privacy level of the Stage instance            |
```

Comment:
`privacy_level` should hyperlink `DOCS_RESOURCES_STAGE_INSTANCE/privacy-level` and other parts that have this property